ViteMaDose/Resources/fr.lproj/Localizable.strings

```diff
@@ -0,0 +1,44 @@
+/* 
+  Localizable.strings
+  ViteMaDose
+
+  Created by Victor Sarda on 24/04/2021.
+  
+*/
+
+// Home
+"home.main_title" = "Trouvez une dose de vaccin facilement et rapidement";
+"home.main_title.first_highlighted_text" = "facilement";
+"home.main_title.second_highlighted_text" = "rapidement";
+"home.select_county" = "Sélectionner un département...";
+"home.recent_search" = "Recherche récente";
+"home.last_stats" = "Dernières statistiques";
+"home.open_map" = "Ouvrir la carte des lieux ";
+"home.partners" = "Trouvez vos rendez-vous avec";
+"home.stats.all_locations" = "Lieux trouvés en France";
+"home.stats.all_availabilities" = "Créneaux disponibles";
+"home.stats.locations_with_availabilities" = "Lieux avec rendez-vous disponibles";
+"home.stats.available_locations_percentage" = "De lieux avec des disponibilités";
+// County Selection
+"county_selection.main_title" = "Sélectionnez un département";
+"county_selection.main_title.highlighted_text" = "département";
+// Locations
+"locations-list.list_title" = "Liste des lieux";
+"locations.no_results" = "Nous n'avons pas trouvé de doses pour le département %@";
```

Comment:
Consistency: These strings appear to belong to `LocationsList`, make it identifiable.
```suggestion
// Locations List
"locations-list.list_title" = "Liste des lieux";
"locations-list.no_results" = "Nous n'avons pas trouvé de doses pour le département %@";
```